### PR TITLE
Additional configuration for HPR http roaming protocol

### PR DESCRIPTION
**Pull request by ruttenberg**
This adds two fields and an enum definition.
It is only used by Helium Packet Router.

### CODE REVIEW

**Review by dpezely — APPROVED** [review]

File: src/service/config.proto [hunk 1]
```diff
@@ -23,7 +24,11 @@ message protocol_gwmp_mapping_v1 {
 
 message protocol_gwmp_v1 { repeated protocol_gwmp_mapping_v1 mapping = 1; }
 
-message protocol_http_roaming_v1 {}
+message protocol_http_roaming_v1 {
+  flow_type flow_type = 1;
+  uint32 dedupe_timeout = 2;
```
Comment (madninja) [inline src/service/config.proto:29]: declare in what unit in a comment

File: src/service/config.proto [hunk 2]
```diff
@@ -23,7 +24,11 @@ message protocol_gwmp_mapping_v1 {
...
+  bytes path = 3;
```
Comment (madninja) [inline src/service/config.proto:30]: declare what this is? if it's an actual http endpoint/path why is this not a string?

Comment (ruttenberg) [reply src/service/config.proto:30]: I was unaware of "string." I will take a look.

File: src/service/config.proto (hunk 1 above)
Comment (ruttenberg) [reply src/service/config.proto:29]: Will do.

File: src/service/config.proto (hunk 2 above)
Comment (ruttenberg) [reply src/service/config.proto]: I have changed it to "string".

File: src/service/config.proto (hunk 1 above)
Comment (ruttenberg) [reply src/service/config.proto]: Done.

**Review by madninja — APPROVED** [review]

**Pull request merged**